waku/node/peer_manager/waku_peer_store.nim

```diff
@@ -16,14 +16,16 @@ import
 export peerstore, builders
 
 type
+  WakuPeerStore* = ref object
+    store*: PeerStore
```

Comment:
I think this is a great opportunity to make the field private. In the suggestion I'm also renaming the attribute :)
```suggestion
    peerStore: PeerStore
```